tailcall-fixtures/fixtures/configs/flatten_single_field.graphql

```diff
@@ -0,0 +1,23 @@
+schema {
+  query: Query
+}
+
+type Query {
+  foo: Foo
+  not_single: NotSingle
+}
+
+# Type with only one field
+type Foo {
+  bar: Bar
+}
+
+# Type with only one field
+type Bar {
+  a: Int
+}
+
+type NotSingle {
+  f1: Int
+  f2: Int
+}
```

Comment:
Add more tests With some recursive types, and more complex nesting.

NOTE: Ensure the test case remains easy to understand no matter how complex the nesting is.

Reply:
Ok.